README.md

```diff
@@ -12,6 +12,8 @@ This repository contains all the tools you need to start building a modern WordP
 
 First you need to install WordPress locally, using any of the local development environment you prefer. You can use XAMPP, MAMP, WAMP, VVV, Docker or Laravel Valet. You'll also need to have [Node.js](https://nodejs.org/en/), [Composer](https://getcomposer.org/) and [WP-CLI](https://wp-cli.org/) installed.
 
+**Note regarding `Windows` - If you're installing this on `Windows` and you're using `XAMPP`, use `XAMPP` shell (Control Panel -> Shell) rather than `cmd` **
```

Comment:
This isn't rendered as bold. A better approach imo would be to add this as a subheading like: 

#### Windows installation

If you're installing this on Windows, and you are using `XAMPP`, use `XAMPP` shell (Control Panel -> Shell) rather than the Windows command line prompt.

Reply:
Changed this a bit so it no longer needs to be bolded.